IOS_CREDENTIALS_EAS.md

```diff
@@ -121,18 +146,18 @@ expo install expo-dev-client
 
 Make sure `"developmentClient": true,` is set in your `eas.json` development build profile.
 
-## All Builds
-
 ### Run the EAS build command
 ```sh
 eas build --profile <build profile> --platform ios
 ```
 where `<build profile>` refers to the build profile in your `eas.json` file (e.g: "development" vs "release").
 
+**IMPORTANT**: run with `EXPO_NO_CAPABILITY_SYNC` if you want to prevent EAS from syncing capabilities (e.g: overwriting the steps you took in the complex setup).
```

Comment:
this is only needed for the managed credentials, if you are using credentials.json then eas-cli is not touching anything on apple side, just taking credentials as they are